snuba/admin/auth_scopes.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Set
+
+
+class AdminRole(Enum):
+    ADMIN = "admin"
+    MEMBER = "member"
+    MEMBER_READ = "member_read"
+
+
+ADMIN_CATEGORY_RESOURCES = {
```

Comment:
Would it be possible/worth it to generate this list automatically from the migration groups in code?

Reply:
+1, we should not have to add a new scope in the admin tool code each time we add a new migration group. That is something product teams do.